.github/workflows/python-packaging.yml

```diff
@@ -25,13 +25,13 @@ jobs:
     - name: Build sdist
       run: python setup.py sdist
       working-directory: ./apis/python
-    - name: Upload sdist artifact
+    - name: Upload sdist artifact # upload to GitHub Actions storage
```

Comment:
any reason not to make this part of the step name, like "Upload sdist to GitHub Actions storage"?